docs/USAGE.md

```diff
@@ -0,0 +1,276 @@
+    return new ApolloClient({
+        link: new HttpLink({
+            uri: `JAHIA_URL/modules/graphql`,
+            headers: {
+              authorization: `APIToken ${token}`
+            }
+        }),
+        cache: new InMemoryCache(),
+        defaultOptions: {
+            query: {
+                fetchPolicy: 'no-cache',
+            },
+        },
+    })
+}
+```
+
+Or, to retrieve the result of the healthcheck, it would be like that:
+
+```bash
+curl -H "Authorization: APIToken XXXXXXXXXX" JAHIA_URL/modules/healthcheck
+```
+
+## Managing tokens
+
+:::info
+Access to the "My API tokens" page is restricted to users with permission "Personal api tokens"
+:::
+
+Tokens that you create display on the **My API tokens** page. The list shows information such as the name of the token, its access key, and expiration date. You can delete, activate and disable tokens from this page.
+
+![Managing Tokens](./img/managing-tokens-01.webp)
+
+:::infos
+**Note:** Tokens are only visible to users once when they are created. None of the information provided here is sufficient to perform an API call.
+:::
+
+To delete a token:
+
+1. Click **Delete** under **Actions** beside the token that you want to delete.
+2. In the Confirmation dialog, click **Delete Forever**.
+
+Once deleted, a token is gone forever and subsequent API calls using that token are denied.
+
+To activate or disable a token:
+
+Click the **More actions** button under **Actions** and select **Activate** or **Disable**.  
+The status changes to **Active** or **Disabled** for the token. Any API call using a disable token is denied.
+
+## Managing tokens for other users
+
+:::info
+Access to the "User API tokens" page is restricted to users with permission "Server Administration > Admin personal api tokens"
+:::
+
+Although you cannot create tokens for other users, you can delete, activate, and disable tokens that you have permissions to view.
+
+To manage another user’s token:
+
+1. Navigate to **Administration>Users and Roles>User API tokens**.
+   ![Create Personal Api Token 3](./img/create-personal-api-token-03.webp)
+
+2. In the search box, enter a user name to filter the list of users.
+3. Delete, activate, and disable the tokens that you want to update.
+
+## Managing tokens through the GraphQL API
+
+Tokens can also be managed through our GraphQL API. The entry point for such operations are located under `admin.personalAPITokens` (query and mutation).
+
+In addition to features provided in the UI, the GraphQL API also provides features to:
+
+- List existing tokens 
+
+```graphql
+query myListTokensQuery {
+  admin {
+    personalApiTokens {
+      tokens {
+        edges {
+          node {
+            createdAt
+            expireAt
+            key
+            name
+            state
+            updatedAt
+          }
+        }
+      }
+    }
+  }
+}
+```
+
+- Check if a token is valid for authentication (verifyToken)
+
+```graphql
+query myVerifyQuery{
```

Comment:
You can remove the name of the query (myVerifyQuery), it does not add much value.